meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -78,13 +78,10 @@ BBMASK += " \
 # meta-arm: mask op-tee as we have our own recipes (until merged in meta-arm)
 BBMASK += " \
     /meta-arm/meta-arm/recipes-security/optee/optee-test_3.8.0.bb \
-    /meta-arm/meta-arm/recipes-security/optee/optee-test_3.11.0.bb \
     /meta-arm/meta-arm/recipes-security/optee/optee-os_3.8.0.bb \
-    /meta-arm/meta-arm/recipes-security/optee/optee-os_3.11.0.bb \
     /meta-arm/meta-arm/recipes-security/optee/optee-examples_3.8.0.bb \
-    /meta-arm/meta-arm/recipes-security/optee/optee-examples_3.11.0.bb \
     /meta-arm/meta-arm/recipes-security/optee/optee-client_3.8.0.bb \
-    /meta-arm/meta-arm/recipes-security/optee/optee-client_3.11.0.bb \
+    /meta-ti/recipes-security/optee/optee-os_%.bbappend \
```

Comment:
I think this will have other bad side effects on other socs, which is why we had the masks in place.

Can we not remove the 3.11 masks and migrate the optee-os used by the TI target be optee-os-fio (which is our fork)?